src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,40 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__link {
+    color: $main-accent;
+    font-weight: 500;
+    font-style: normal;
+    text-transform: uppercase;
+    text-decoration: none;
+    margin-left: 20px;
+    display: block;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  &__link:hover {
```

Comment:
Use sass nesting here and for all pseudo-elements and pseudo-classes that your are using